dbms/src/main/java/org/polypheny/db/partition/ListPartitionManager.java

```diff
@@ -40,13 +40,13 @@ public class ListPartitionManager extends AbstractPartitionManager {
 
 
     @Override
-    public long getTargetPartitionId( CatalogTable catalogTable, String columnValue ) {
+    public long getTargetPartitionId( LogicalTable table, PartitionProperty property, String columnValue ) {
         long unboundPartitionId = -1;
         long selectedPartitionId = -1;
 
         // Process all accumulated CatalogPartitions
-        for ( CatalogPartition catalogPartition : Catalog.getInstance().getPartitionsByTable( catalogTable.id ) ) {
-            if ( unboundPartitionId == -1 && catalogPartition.isUnbound ) {
+        /*for ( AllocationEntity entity : targetEntities ) {
+            if ( targetProperty.isUnbound ) {
```

Comment:
How does partitioning currently work with this being commented out?

Reply:
Uncommented and fixed, takes now correct partition for list